packages/trees/frontend/sirius-components-trees/src/trees/FilterBar.tsx

```diff
@@ -33,11 +35,18 @@ const useFilterBarStyles = makeStyles((theme: Theme) => ({
     fontSize: theme.typography.fontSize,
   },
 }));
-export const FilterBar = ({ onTextChange, onFilterButtonClick, onClose }: FilterBarProps) => {
+export const FilterBar = ({
+  onTextChange,
+  onTextClear,
+  onFilterButtonClick,
+  onClose,
+  text,
+  option,
+}: FilterBarProps) => {
   const classes = useFilterBarStyles();
 
   const initialState: FilterBarState = {
-    filterEnabled: false,
+    filterEnabled: option?.filterEnabled,
```

Comment:
Shouldn't it be `option ? option.filterEnabled : false,` or `option?.filterEnabled || false,`?

Reply:
I want the default value to be false, so if `option` is undefined, `option?.filterEnabled` will be computed as false. I don't think I need to specified ` || false`

Reply:
Unless I'm missing something if `option` is `undefined`, then `option?.filterEnabled` will be computed as `undefined`.
The attribute `filterEnabled` type is `boolean`, not `boolean | undefined`, so I think `|| false` is necessary.